README.md

```diff
@@ -90,17 +90,25 @@ jobs:
     steps:
       # Checkout the repository sources (GITHUB_WORKSPACE)
       - name: Checkout
-        uses: actions/checkout@v3.1.0
+        # You may instead pin an action SHA, like this: 
+        # uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
+        uses: actions/checkout@v3.5.3
+        with:
+          # The default depth of 1 commit is not enough for some scans 
+          fetch-depth: 0
         
       - name: Xygeni-Scanner
-        uses: xygeni/xygeni-action@v2.0
+        # Pinned: xygeni/xygeni@4e94e5ea737f5ebfedac8b1a5b75d8c60c21932d
+        uses: xygeni/xygeni-action@v2.1
```

Comment:
Probably v.3.2.0 should be used here. (SHA 4e94e5ea737f5ebfedac8b1a5b75d8c60c21932d).

https://github.com/xygeni/xygeni-action/releases/tag/v3.2.0

Reply:
Added